src/components/i18n.js

```diff
@@ -22,22 +19,38 @@ const getMyCampusLanguage = () => {
     else {
         return (chrome.runtime.getManifest()).default_locale
     }
```

Comment:
What if the user has a different language like Italian set as his default?

Reply:
It's only the language of MyCampus, it doesn't care about the browser language.

Reply:
`(chrome.runtime.getManifest()).default_locale` returns the language of the MyCampus site?

Reply:
no, it returns the value of the default locale in the manifest.json